src/endpoints/collections/collection.module.ts

```diff
@@ -11,6 +12,7 @@ import { CollectionService } from "./collection.service";
     forwardRef(() => VmQueryModule),
     forwardRef(() => TokenModule),
     forwardRef(() => AssetsModule),
+    PluginModule,
```

Comment:
A circular dependency between modules. Use forwardRef() to avoid it.
 forwardRef(() => PluginModule)

Reply:
Done